README.md

```diff
@@ -93,6 +93,22 @@ This site exposes a public JSON API powered by [Next.js API routes](https://next
 - [`/api/r/:emoji`](https://scrapbook.hackclub.com/api/r/hardware) – Get all posts tagged with a specific emoji
 - [`/:username.png`](https://scrapbook.hackclub.com/sampoder.png) – Get a user's avatar as an image URL
 
+## Running Locally
+
+1. Clone this repository
+   - `git clone git@github.com:hackclub/scrapbook.git; cd scrapbook`
```

Comment:
```suggestion
   - `git clone https://github.com/hackclub/scrapbook.git && cd scrapbook`
```
If a user doesn't have their SSH key on GitHub, they would have to add that before doing this step; would it just be easier to change the instructions to clone using HTTPS?

<img width="364" alt="Screen Shot 2022-03-02 at 10 24 01" src="https://user-images.githubusercontent.com/72365100/156424420-6089afbf-a027-4f7a-b4fb-d131e500303d.png">

Reply:
@exu3 Smart! Done in [e80b1bb](https://github.com/hackclub/scrapbook/pull/280/commits/e80b1bb18329133ad27406a588bc5822aad3ded4)